zone/gm_commands/setanim.cpp

```diff
@@ -2,9 +2,24 @@
 
 void command_setanim(Client *c, const Seperator *sep)
 {
-	int arguments = sep->argnum;
+	const auto arguments = sep->argnum;
 	if (!arguments || !sep->IsNumber(1)) {
-		c->Message(Chat::White, "Usage: #setanim [Animation ID (IDs are 0 to 4)]");
+		c->Message(Chat::White, "Usage: #setanim [Animation ID]");
+
+		uint32 animation_number = 1;
```

Comment:
This is where inputs and stuff are sent on dialed input.

Reply:
Roger